### PR TITLE
US-14895: updated the header text for CoB

### PR DESCRIPTION
**Pull request by lcart97**
Update the Header for the Change of Bank flow.

### CODE REVIEW

File: assets/i18n/cy.json [hunk 1]
```diff
@@ -374,7 +374,7 @@
   "POE_FOR_ELDEST_CHILD": " ar gyfer y plentyn hynaf neu’r unig blentyn",
   "POE_FOR_ADDITIONAL_CHILD": " ar gyfer pob plentyn ychwanegol",
 
-  "CHB_HOMEPAGE_HEADING": "Budd-dal Plant",
+  "CHB_HOMEPAGE_HEADING": "Newidiwch fanylion eich taliad Budd-dal Plant",
```
Comment (lcart97) [inline assets/i18n/cy.json:377]: To be confirmed

**Review by akshaykalaskar1 — APPROVED** [review]

**Review by AkhilaHMRC — CHANGES_REQUESTED** [review]
Dont change the existing value, this will impact the service name in other CHB journeys, this is not desired. The service name is applicable only for COB journey. Create a new key for the COB service name in lang json and use that key in COB component for service name

File: assets/i18n/cy.json [hunk 2]
```diff
@@ -404,8 +404,10 @@
   "ERROR": "Gwall",
   "RETURN_TO_CHANGE_CLAIM": "Dychwelyd i newid yr hawliad",
   "NO_ACTION": "No action",
+
+  "COB_SERVICE_HEADING": "Newid eich manylion taliadau Budd-dal Plant",
```
Comment (lcart97) [inline assets/i18n/cy.json:408]: Confirmed by Asief

**Comment by lcart97** [comment]
@AkhilaHMRC - I have confirmed the welsh translation and also created a new value in the 2 jsons.

File: src/samples/ChangeOfBank/ChangeOfBank.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ export default function ChangeOfBank() {
   const { hmrcURL } = useHMRCExternalLinks();
   const setAuthType = useState('gg')[1];
 
-  registerServiceName(t('CHB_HOMEPAGE_HEADING'));
+  registerServiceName(t('CHILD_BENEFIT'));
```
Comment (AkhilaHMRC) [inline src/samples/ChangeOfBank/ChangeOfBank.tsx:35]: update this same as the service name for COB

**Review by AkhilaHMRC — CHANGES_REQUESTED** [review]

File: src/samples/ChangeOfBank/ChangeOfBank.tsx (hunk 3 above)
Comment (lcart97) [reply src/samples/ChangeOfBank/ChangeOfBank.tsx:35]: to the same as the heading? i.e. 
COB_SERVICE_HEADING

Comment (AkhilaHMRC) [reply src/samples/ChangeOfBank/ChangeOfBank.tsx]: yes

File: assets/i18n/en.json [hunk 4]
```diff
@@ -413,8 +413,10 @@
   "ERROR": "Error",
   "RETURN_TO_CHANGE_CLAIM": "Return to change claim",
   "NO_ACTION": "No action",
+
+  "COB_SERVICE_HEADING": "Change your Child Benefit payment details",
   "COB_PAGE_HEADING": "Change the account you use for Child Benefit payments",
-  "COB_PAGE_WARRNING": "Child Benefit due  within 3 working days will go into your existing account.",
+  "COB_PAGE_WARNING": "Child Benefit due  within 3 working days will go into your existing account.",
```
Comment (gautamkumar-hmrc) [inline assets/i18n/en.json:419]: extra space after due

**Review by gautamkumar-hmrc — APPROVED** [review]

**Review by AkhilaHMRC — APPROVED** [review]

**Pull request merged**